app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/individual/HealthRecordsAdapter.kt

```diff
@@ -77,7 +79,7 @@ class HealthRecordsAdapter(
                     isUpdateRequested = false
                 } else {
                     holder.binding.progressBar.visibility = View.INVISIBLE
-                    holder.binding.ivRightArrow.visibility = View.VISIBLE
+                    holder.binding.ivRightArrow.visibility = if (canDeleteRecord) View.GONE else View.VISIBLE
                 }
```

Comment:
Can disable navigation in edit mode by using canDeleteRecord variable in the item click listener condition.